src/node_ref.rs

```diff
@@ -1979,10 +2006,11 @@ where
     /// assert_eq!(a.node(dfs_indices[2]).data(), &3);
     /// assert_eq!(a.node(dfs_indices[3]).data(), &7);
     /// ```
-    fn indices<T>(&self) -> impl Iterator<Item = NodeIdx<V>>
+    fn indices<'t, T>(&self) -> impl Iterator<Item = NodeIdx<V>> + 't
     where
-        T: Traverser<OverData>,
-        V: 'static,
+        T: Traverser<OverData> + 't,
+        <T as TraverserCore>::Storage<V>: 't,
```

Comment:
Couldnt this line be omitted as (to my knowledge) it is already inherited by the above `T: /*...*/ + 't`?